tests/manage/mcg/test_mcg_resources_disruptions.py

```diff
@@ -297,3 +311,62 @@ def test_db_scc(self, teardown):
         ), "Invalid scc"
         # Check the NB status to verify the system is healthy
         self.cl_obj.wait_for_noobaa_health_ok()
+
+    @tier4b
+    @skipif_bm
+    @skipif_aws_i3
+    @skipif_vsphere_ipi
+    @skipif_ibm_power
+    @bugzilla("2029690")
+    @pytest.mark.polarion_id("")
+    def test_mcg_nw_failure(
```

Comment:
We already have existing test test_all_worker_nodes_short_network_failure [here](https://github.com/red-hat-storage/ocs-ci/blob/master/tests/manage/z_cluster/nodes/test_worker_nodes_network_failures.py) 
The steps looks almost similar, should we simply update the existing test to create bucket after network failure?

Reply:
ack will create a new pr.